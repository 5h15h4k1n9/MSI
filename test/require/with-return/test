earlyReturn(0)
counter
earlyReturn(100)
exit
