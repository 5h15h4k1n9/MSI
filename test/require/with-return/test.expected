(void) void
(int) 0
Reverted with: Too big
Bye-bye!
